com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/StackFrameUtility.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package com.microsoft.java.debug.core;
+
+import com.sun.jdi.AbsentInformationException;
+import com.sun.jdi.IncompatibleThreadStateException;
+import com.sun.jdi.ReferenceType;
+import com.sun.jdi.StackFrame;
+
+public final class StackFrameUtility {
```

Comment:
StackFrameUtils for the same name conventions

Reply:
Actually  we have DebugUtility here, not use where is *Utils coming from.